src/shadowStyles.js

```diff
@@ -5,12 +5,21 @@
 // Main library
 (function(){
   "use strict";
-
-  function initialize(document, window) {
+    
     // A buffer must be made to bridge the elements to negated CSS selectors
     var BUFFER_ATTR = 'css-negate';
-    var SHADOW_ATTR = 'shadow'
+    var SHADOW_ATTR = 'shadow';
     var UNIQUE_ID_LENGTH = 5;
+    /* 
+      Allowing the consumer to define the
+      dependency if they are within the commonJS environment.
+      A similar approach could be used for other dependencies
+      to encourage modular development.
+      The docs would mention that commonJS users would need `npm install` it into
+      their project if they need to support browsers that
+      don't support `MutationObserve` natively.
+    */
+    var MutationObserver = (typeof(module) !== 'undefined') ? require('mutation-observer') : window.MutationObserver;
```

Comment:
This line should fit much better within 80 characters and there should be changes to the readme to match this.

Why can't the `mutation-observer` package be included in the `dependencies` in `package.json`?